bokehjs/src/coffee/models/plots/plot.coffee

```diff
@@ -88,32 +115,66 @@ class Plot extends LayoutDOM.Model
     if not @height?
       @height = @plot_height
 
+  Object.defineProperty(this.prototype, "plot_canvas", { get: () -> @_plot_canvas })
```

Comment:
The way you implemented this suggests that you wanted to make `_plot_canvas` private and immutable from outside (as much it is possible in JS). What you see here is the correct implementation of that intent. Of course we can initialize `plot_canvas` directly, but that has different semantics. Which do you actually want?